packages/manager/apps/key-management-service/src/pages/dashboard/index.tsx

```diff
@@ -18,6 +18,9 @@ import Breadcrumb from '@/components/Breadcrumb/Breadcrumb';
 import { ROUTES_URLS } from '@/routes/routes.constants';
 import { BreadcrumbItem } from '@/hooks/breadcrumb/useBreadcrumb';
 import { getOkmsResourceQueryKey } from '@/data/api/okms';
+import { OKMS } from '@/types/okms.type';
+
+export const OkmsContext = createContext<OKMS>(null);
```

Comment:
I originally put it in a separate file in the hooks folder, but i moved it back to this file because it's a context that is intended to be used only in children of a kms dashboard (tabs, modals) to prevent prop drilling.

Maybe you're right and it should be moved back in the hooks folder?

Reply:
Checked the best practices : 
This folder is dedicated to defining the custom hooks that are reusable in the context of the application where it is defined.

And with : 
Each hook must be defined inside a sub-folder with name of the sub-folder being the name of the hook. This ensures to group the related files like spec, constants and hook definition files in single place.

We're not following yet this structure as we don't have subfolder, but this hook could be placed into a hooks/okms folder 

In fact i would let it as is it and plan a refacto for the whole hooks folder at some point to follow the best practices.